src/StreamDeck/Transport/LibUSBHIDAPI.py

```diff
@@ -34,10 +34,10 @@ def _load_hidapi_library(self, library_search_list):
 
             if self.HIDAPI_INSTANCE:
                 return self.HIDAPI_INSTANCE
-
+            
             for lib_name in library_search_list:
                 try:
-                    self.HIDAPI_INSTANCE = ctypes.cdll.LoadLibrary(lib_name)
+                    LibUSBHIDAPI.Library.HIDAPI_INSTANCE = ctypes.cdll.LoadLibrary(lib_name)
```

Comment:
It's safe to read from a class variable with "self.", but trying to write to one will instead assign a variable of that name to the instance.